### ISSUE TITLE
redis-cluster , external access.

### ISSUE THREAD

**Issue by sudo-ankitmishra** [bitnami/charts#8876]
Hi.
I am trying this command 
**helm install redis bitnami/redis-cluster --set 'cluster.nodes=6,cluster.externalAccess.enabled=true'**
on minikube and the response I got is
**Error: INSTALLATION FAILED: template: redis-cluster/templates/svc-cluster-external-access.yaml:15:24: executing "redis-cluster/templates/svc-cluster-external-access.yaml" at <.Release.Namespace>: can't evaluate field Release in type int**
Can some please help me with this. 
I also tried changing the values.yaml according to the doc, but unable to get a cluster up and running.

helm install command runs without any issues if I do not pass any parameter.
Thanks

**Comment by javsalgar** [comment]
Hi,

I found the cause of the issue. It is because of this PR https://github.com/bitnami/charts/pull/8821

File: redis-cluster/templates/svc-cluster-external-access.yaml
```
  namespace: {{ .Release.Namespace | quote }}
```

It should be

```
  namespace: {{ $root.Release.Namespace | quote }}
```

Would you like to submit a PR with the fix? If not we will work on it. 

Thank you so much for reporting

**Comment by sudo-ankitmishra** [comment]
Hello @javsalgar.
Thank you for the information.

Is there any workaround for now, such that I can use the redis-cluster chart, like the chart version.
Thanks.

**Comment by sudo-ankitmishra** [comment]
Thanks for help, issue if fixed.

**Comment by Shujja1746** [comment]
@javsalgar

The lastest version for external redis is still not working.

Can you let me know when it can be available.

Thanks

**Comment by javsalgar** [comment]
Hi,

Could you provide the exact issue and the values you are using?

**Comment by Shujja1746** [comment]
Hi 

I am using using the latest values file 
https://github.com/bitnami/charts/blob/master/bitnami/redis-cluster/values.yaml

When we do external its not working on azure aks.

![image](https://user-images.githubusercontent.com/100185414/158414056-c2b15984-0d13-48bc-be82-edc0df813860.png)

cluster is showing down
Warning: Using a password with '-a' option on the command line interface may not be safe.
20.94.25.85:6379> SET FOO BAR
(error) CLUSTERDOWN Hash slot not served
20.94.25.85:6379>

**Comment by javsalgar** [comment]
Hi,

I'm not sure I fully understand. You mention that there are issues with the PVCs, but the issue lies when using externalAccess, could you specify which exact values you set for enabling external access? Did you set the list of LoadBalancer IPs? Did you use autoDiscovery?

**Comment by Basitkhan23** [comment]
Hello,

I have faced the an issue with 11.4.0 version. Not sure If I was missing something while installing the cluster or not. 

`Error: INSTALLATION FAILED: template: redis-cluster/templates/svc-cluster-external-access.yaml:18:16: executing "redis-cluster/templates/svc-cluster-external-access.yaml" at <include "common.names.namespace" .>: error calling include: template: redis-cluster/charts/common/templates/_names.tpl:63:20: executing "common.names.namespace" at <.Release.Namespace>: can't evaluate field Release in type int`

But after I installed 11.2.0, it didn't throw any errors.
Looks like "{{ $.Release.Namespace | quote }}" has been changed in file "svc-cluster-external-access.yaml" 11.2.2 version. Not sure if that does effect the installation or not.
